common/zfs/util.go

```diff
@@ -103,14 +104,23 @@ func (zfsDevicePoolConfig *ZfsNodesDevicePoolConfig) ConfigureZfsNodesWithDevice
 
 func SetupZfsNodes(poolName string, size int64) (ZfsNodesDevicePoolConfig, error) {
 	var zfsNodeConfig ZfsNodesDevicePoolConfig
-	loopDevice := e2e_agent.LoopDevice{
-		Size:   size,
-		ImgDir: "/tmp",
-	}
 	workerNodes, err := ListZfsNode(common.NSOpenEBS())
 	if err != nil {
 		return zfsNodeConfig, fmt.Errorf("failed to list zfs worker nodes, error: %v", err)
 	}
+	if len(workerNodes) == 0 {
+		return zfsNodeConfig, fmt.Errorf("zfs worker nodes not found")
+	}
+	var imgDir string
+	if strings.Contains(workerNodes[0], "kind-") {
```

Comment:
Please add comments describing what is being detected and assumptions/expectations of the cluster/node setup

Reply:
Added comments https://github.com/openebs/openebs-e2e/pull/65/commits/560dcf6648ec842d8c1f104a8d58f9dc1ed8f129